<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartPage</class>
 <widget class="QWidget" name="StartPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,5,0,0">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>104</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>104</number>
   </property>
   <property name="bottomMargin">
    <number>45</number>
   </property>
   <item>
    <widget class="QScrollArea" name="system_information_scroll_area">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="sys_scroll_content">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1037</width>
        <height>156</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="sys_info_frame_h_layout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="ep_widgets_container" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="ep_filters_container_layout">
        <property name="spacing">
         <number>12</number>
        </property>
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="select_configs_label">
          <property name="text">
           <string>Select one or more configurations to run</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="ep_scroll_area">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="m_scroll_area_widget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1019</width>
           <height>342</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="eps_v_layout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="disclaimer_text_label">
       <property name="text">
        <string>Configuration choices will affect how the benchmark runs and performs.
Performance may also vary due to factors like system load and thermals.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_download_only_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>49</height>
        </size>
       </property>
       <property name="text">
        <string>Download Only</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_start_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>289</width>
         <height>49</height>
        </size>
       </property>
       <property name="text">
        <string>Run Benchmark Tests (0)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
